src/Program.ts

```diff
@@ -1194,12 +1195,59 @@ export class Program {
      * This does not write anything to the file system.
      */
     public getTranspiledFileContents(pathAbsolute: string) {
-        let file = this.getFile(pathAbsolute);
-        let result = file.transpile();
+        return this._getTranspiledFileContents(
+            this.getFile(pathAbsolute)
+        );
+    }
+
+
+    /**
+     * Internal function used to transpile files.
+     * This does not write anything to the file system
+     */
+    private _getTranspiledFileContents(file: BscFile, outputPath?: string): FileTranspileResult {
```

Comment:
Perhaps I should come up with a better name for this? I don't like the underscore. I don't want to use the word "internal" either though.